### ISSUE TITLE
add a demo that benchmarks 2 R optimizers and shows postproc

### ISSUE THREAD

**Issue by berndbischl** [berndbischl/rcoco#32]
we need to decide where to put this.

vignette might be best

**Comment by jakobbossek** [comment]
Agree

**Comment by berndbischl** [comment]
i have begun here

bechmark_demo.R (in main dir)

the plots look a bit strange (just 1 algo visible?)
and i am getting some warning in postproc call

too tired now. over and out

**Comment by jakobbossek** [comment]
Yes I experienced the same behaviour of the postprocessing. Going to have look at this later.

**Comment by jakobbossek** [comment]
We need to call `python -m cocopp result_folder/algo1 result_folder/algo2 ... result_folder/algon` now in order to generate comparisson data.

**Comment by jakobbossek** [comment]
Added a first draft of the vignette (not finished).
